lecture2.tex

```diff
@@ -99,25 +114,23 @@ \section{Kombinatoriska bevis}
   \begin{marginfigure}
     \includegraphics{graphics/pascals_triangle.png}
   \end{marginfigure}
-  $$\binom{n}{k} = \binom{n-1}{k-1} + \binom{n-1}{k}.$$
+  $$\binom{n}{k} = \binom{n-1}{k-1} + \binom{n-1}{k}.$$ 
 
   \begin{proof}[Algebraiskt bevis]
     \begin{align*}
       \binom{n-1}{k-1} + \binom{n-1}{k} &= \frac{(n-1)!}{(k-1)!((n-1)-(k-1))!} + \frac{(n-1)!}{k!((n-1)-k)!}\\
       &= (n-1)!\left(\frac{k}{k!(n-k)!} + \frac{(n-k)}{k!(n-k)!}\right)\\
       &= (n-1)!\frac{k + (n - k)}{k!(n-k)!}\\
       &= \frac{n!}{k!(n-k)!} = \binom{n}{k}.
-    \end{align*}    
+    \end{align*}   
   \end{proof}
 
   \begin{proof}[Kombinatoriskt bevis]
     Låt $X$ vara en mängd av storlek $n$. Vi vet att $\binom{n}{k}$ räknar antalet delmängder av storlek $k$ till $X$. Låt oss komma på ett annat sätt att räkna antalet delmängder av storlek $k$, och se att det ger oss höger led.
-
-    Välj ett godtyckligt element $x \in X$. För varje delmängd $A$ till $X$ så innehåller den antingen $x$, eller så gör den inte det. För att skapa oss en delmängd $A$ av storlek $k$ som innehåller $x$ lägger vi först in $x$ i $A$, och sedan lägger vi till ytterligare $k-1$ element från de återstående $n-1$ elementen. Antalet sätt att göra det vet vi är precis $\binom{n-1}{k-1}$
-
-    Om vi i stället vill ha en delmängd $A$ som \emph{inte} innehåller $x$ så kan vi fritt välja $k$ element av de återstående $n-1$ elementen. Antalet sätt att göra det på vet vi är $\binom{n-1}{k}$.
-
-    Så additionsprincipen säger oss att antalet sätt att välja en delmängd som innehåller $x$ eller inte innehåller $x$ -- vilket ju är alla delmängder -- måste vara summan, alltså $\binom{n-1}{k-1} + \binom{n-1}{k}$, såsom önskat.
+    Välj ett godtyckligt element $x \in X$. Vi låter V beteckna alla delmängder till X som innehåller $x$, dvs $V=\{A\subseteq X:x\in A\} =\{ \{ x\} \cup B | B \subseteq X \backslash \{ x\} , |B|=k-1\}$, och vi låter H beteckna alla delmängder till X som inte innehåller x, dvs $H=\{A\subseteq X:x\notin A\} =\{A \subseteq X \backslash \{ x\} , |A|=k\}$. Vi ser att V och H är disjunkta och att unionen av dem är potensmängden av X. Idén är att räkna antalet delmängder av storlek k som tillhör V och H, och sedan använda additionsregeln för att räkna antalet delmängder av storlek k till X. 
```

Comment:
**Alternativt:**
Vi låter:
`\par\begin{itemize}[leftmargin=.75cm]
  \item $V = \left\{A\subseteq X\::\: x\in A\right\} = \left\{\left\{x\right\}\cup B\::\: B\subseteq X\backslash\left\{x\right\}\::\: \left|B\right| = k-1\right\}$
    \par\bigskip
\item $H = \left\{A\subseteq X\::\: x\notin A\right\} = \left\{A\subseteq X\backslash\left\{x\right\}\::\: \left|A\right| = k\right\}$
  \par\bigskip
\item $V\cap H = \O$
  \par\bigskip
\end{itemize}`

(där `$\O$` är tomma-mängden)